src/components/GameComponent.tsx

```diff
@@ -1,25 +1,33 @@
 import React, { useEffect } from 'react';
 import Phaser from 'phaser';
-import HelloWorldScene from '../phaser-game/scenes/HelloWorldScene';
+import { GridEngine } from 'grid-engine';
+import FortNerf from '../phaser-game/scenes/FortNerf';
 
 const config = {
   type: Phaser.AUTO,
   // scale: {
   //   mode: Phaser.Scale.ScaleModes.RESIZE,
   //   width: window.innerWidth,
-  //   height: window.innerHeight,
+  //   height: 600,
   // },
```

Comment:
Are these jus notes for now? Otherwise, probably don't need 7-11 comments anymore.